test/PoolManager.t.sol

```diff
@@ -814,6 +826,8 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         });
         swapRouter.swap(key, params, testSettings, ZERO_BYTES);
 
+        // input is 1000 for output of 998 with this much liquidity available
+        // plus a fee of 1.23% on unspecified (output) => (998*123)/10000 = 12
```

Comment:
just so ultra clear - this test is a hook bearing fee on unspecified ? the test name just says custom accounting, but then the hook on the pool is a fee bearing hook